hnn_core/network.py

```diff
@@ -488,3 +479,68 @@ def plot_spikes(self, ax=None, show=True):
         if show:
             plt.show()
         return ax.get_figure()
+
+    def build_in_neuron(self):
```

Comment:
if we do not expect the user to access this function, we should make it a private function

Reply:
It gets called from parallel.py. Still private?

Reply:
Oops. This function gets called in test_network.py. I'll add docstrings. clear_neuron_objects() is the one that I was thinking of that gets called from parallel.py. I'll convert it to private naming

Reply:
_build is now private